modules/core/src/main/java/org/apache/ignite/internal/processors/cache/ClearCachesTaskArg.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2024 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.List;
+import org.apache.ignite.internal.util.tostring.GridToStringInclude;
+import org.apache.ignite.internal.util.typedef.internal.S;
+import org.apache.ignite.internal.util.typedef.internal.U;
+import org.apache.ignite.internal.visor.VisorDataTransferObject;
+
+/** Argument for {@link ClearCachesTask}. */
+public class ClearCachesTaskArg extends VisorDataTransferObject {
```

Comment:
Please use `IgniteDataTransferObject` instead of `VisorDataTransferObject`.

Reply:
done

Reply:
Hi @kingnoskcaj ,

Could you please fix the same issue here: https://github.com/gridgain/gridgain/pull/3139/files#diff-d42cf87402346a759eb2061f6a5d0fbd79020810dcb163e60937c50379770142R29

Reply:
Can do!